zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZeppelinSparkClusterTest.java

```diff
@@ -154,6 +162,11 @@ private void waitForRunning(Paragraph p) {
 
   @Test
   public void scalaOutputTest() throws IOException, InterruptedException {
+    if (!isHadoopVersionMatch()) {
```

Comment:
IMO this would be better by using JUnit Assumption APIs, e.g. `assumeThat(isHadoopVersionMatch(), "Hadoop version mismatch, skipping the test")`. No need of the `return;`

Reply:
Fixed